test/spec/auctionmanager_spec.js

```diff
@@ -232,6 +232,15 @@ describe('auctionmanager.js', function () {
       assert.deepEqual(response, expected);
     });
 
+    it('should suppress acat if undefined', function () {
+      bid.meta.primaryCatId = ''
+      let expected = getDefaultExpected(bid);
+      delete expected.hb_acat;
+      let response = getKeyValueTargetingPairs(bid.bidderCode, bid);
+      assert.deepEqual(response, expected);
+      bid.meta.primaryCatId = 'IAB-test';
```

Comment:
this should be in the test teardown - but really, I think this shows that on test setup, line 223, we should be `deepClone(DEFAULT_BID)`.

the problem with doing it this way is that if the test fails for any reason in the future, for example we change the business logic around acat, the teardown does not run and instead of one failure you'll get many false failures from subsequent tests that expect `bid.meta` to be "clean".

Reply:
@dgirardi thanks, that makes total sense regarding how my test was setup.  I just refactored my test to use `deepClone` instead of using the `bid` var that was in use for multiple tests.